src/components/displayDocuments/backButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { MdArrowBack } from "react-icons/md";
+
+const BackButtonContainer = styled.div`
+  background: #feeee7;
+  border-radius: 12px 0px 4px;
+  width: 25px;
+  height: 25px;
+`;
+
+interface BackButtonProps {
+  className?: string;
+  handleClick: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
+}
+
+export const BackButton: React.FunctionComponent<BackButtonProps> = ({ handleClick }) => {
```

Comment:
for consistency:
- both button must have the same size
- rather than an arrow I would use something that makes me think about certificate/document. I think MdDescription fits better. Can ask to @sebastianquek

Reply:
if i keep the same size, the qr code doesn't work as part of it is covered, if i align them to the center. how should i handle it?